types/scte224v20180501/sctestructures.go

```diff
@@ -144,16 +144,16 @@ func (ap *Apply) GetPriority() uint {
 
 //Table 9
 type Remove struct {
-	XMLName xml.Name `xml:"http://www.scte.org/schemas/224 Remove"`
-	Policy  *Policy  `xml:"http://www.scte.org/schemas/224 Policy",omitempty`
```

Comment:
notice the " is before the omitempty statement which probably is causing problems.

Reply:
does it really make sense to have the xml namespace in the json tag?  from line 20 `json:"xml:base,omitempty"`